### DISCUSSION TITLE
Welcome to fundamental-styles Discussions!

### DISCUSSION THREAD

**Discussion by droshev** [SAP/fundamental-styles#1950] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fopsdev** [comment]
Hi there

I'm using web technologies (well, not only) to create affordable enterprise webapps for small and medium sized companies.
Having tried existing framework they ended up beeing too heavy for my customers needs. So i ended up creating my own set of custom elements using a reactive approach with a custom state store and more functionality heavely inspired by https://overmindjs.org/

I would like to use fundamental-styles inside those custom elements because most of my customers are using SAP as their ERP.

Thanks a lot for providing such a complete set of css components!

**Reply by droshev** [reply]
good luck
